Automattic/sensei
# Add theme compatibility to the learner profile page

Partial solution to #2154

Using the same abstract from #2235, this adds the support for the learner profile in unsupported themes. In addition, this also fixes an issue with the My Messages link floating strangely when viewing your own profile page. 

## Testing
- Sign up a user for multiple courses or no courses at all.
- Make sure the course profiles are enabled in Sensei's settings (disabled by default).
- Visit the learner profile page for multiple users (`/learner/{user_nicename}`). 
- Ensure that the page looks right on themes that support Sensei and themes that do not support Sensei.
- For themes that do not support Sensei, ensure that:
  - The title is not duplicated.
  - The pagination does NOT show up at the bottom.
